CHANGELOG.md

```diff
@@ -6,7 +6,11 @@
 ## Version X.Y.Z
 
 Date:            XX/YY/ZZZZ
-Contributors:    @RMeli, @takluyver
+Contributors:    @RMeli, @takluyver, @Jnelen
+
+### Added
+
+* Functionality to manually select the backend [PR  #107 | @Jnelen, @RMeli]
```

Comment:
```suggestion
* Functionality to manually select the backend [PR  #107 | @Jnelen]
```
While you are at it, can you please also add a line under "changed", mentioning that the molecular graph is now cached per backend?

This is really the last thing! =)